production/loki-mixin/config.libsonnet

```diff
@@ -11,6 +14,20 @@
 
     // The label used to differentiate between different clusters.
     per_cluster_label: 'cluster',
+    per_namespace_label: 'namespace',
+    per_job_label: 'job',
+
+    // Grouping labels, to uniquely identify and group by {jobs, clusters}
+    job_labels: [$._config.per_cluster_label, $._config.per_namespace_label, $._config.per_job_label],
+    cluster_labels: [$._config.per_cluster_label, $._config.per_namespace_label],
```

Comment:
just to avoid confusion lets make the names as explicit as possible, I would assume that `job_labels` and `cluster_labels` select only `job` or `cluster`, not a combination of those plus other labels.

Reply:
@cstyan,
Sorry for the late reply, need to have a look at my notification settings... 

I do not necessarily disagree with that, my thoughts behind it was that i wanted it to be familiar for user to use it, and since Mimir mixins does it like that... https://github.com/grafana/mimir/blob/main/operations/mimir-mixin/config.libsonnet#L179-L182

Reply:
I don't have a very strong preference either way, but I think the simplified naming is for historical reasons and we've just tacked on additional labels as we went without changing the name. Your reasoning is valid as well, I'll leave the end decision here to you.